configurations/host/networking.nix

```diff
@@ -0,0 +1,27 @@
+{...}: {
+  networking = {
+    enableIPv6 = false;
+    firewall.allowedUDPPorts = [67]; # DHCP
+    useNetworkd = true;
+  };
+  systemd.network = {
+    netdevs."virbr0".netdevConfig = {
+      Kind = "bridge";
+      Name = "virbr0";
+    };
+    networks."virbr0" = {
```

Comment:
We get `IPv6` on `virbr0` even when it's disabled on line 3 here
```
[ghaf@ghaf-host:~]$ ifconfig virbr0
virbr0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet6 fe80::289d:b5ff:fe12:1614  prefixlen 64  scopeid 0x20<link>
        ether 2a:9d:b5:12:16:14  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 64  bytes 5239 (5.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

Reply:
Looks like you can still assign ipv6 addresses to interfaces.

Hovewer the setting enableIPv6 = false; sets
```
# sysctl net.ipv6.conf.all.disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 1
```

So the actual IPv6 based traffic does not move anywhere